File: SampleCodeAPI/Model/LoaiDaoTaoModel.cs
namespace SampleCodeAPI.Model
{
    public class LoaiDaoTaoModel
    {
        public int id { get; set; }
        public string MaLoaiDT { get; set; }
        public string TenLoaiDT { get; set; }
        public string? TenTiengAnh { get; set; }
        public string? NoiDung { get; set; }
        public long? SoThuTu { get; set; }
        public string GhiChu { get; set; }
        public bool? IsDel { get; set; }
        public int? DeletedBy { get; set; }
        public int? UpdatedBy { get; set; }
        public int? CreatedBy { get; set; }
        public DateTime? DeletedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
        public DateTime? CreatedDate { get; set; }
        

    }
}

File: Frontend/Angular/src/app/pages/danhmuc/loai-dao-tao/model/loai-dao-tao.model.ts
import { BaseModel } from "src/app/_metronic/core/models/_base.model";

export class LoaiDaoTaoModel extends BaseModel {
    id: number;
    MaLoaiDT: string;
    TenLoaiDT: string;
    TenTiengAnh: string;
    NoiDung: string;
    SoThuTu: number | null | string;
    GhiChu: string;
    IsDel?: boolean;
    CreatedBy?: string;
    CreatedDate?: Date | string;

    clear() {
        this.id = 0;
        this.MaLoaiDT = '';
        this.TenLoaiDT = '';
        this.TenTiengAnh = '';
        this.NoiDung = '';
        this.SoThuTu = 0;
        this.GhiChu = '';
        this.IsDel = false;
        this.CreatedBy = '';
        this.CreatedDate = new Date();
    }
}
